libs/parsers/src/lib/expression/parser.ts

```diff
@@ -43,14 +50,14 @@ export class ExpressionParser {
     },
   ];
 
-  public static parseExpression(expression: string): Expression {
+  public static parseExpression(expression: string, isAliasPathParsingEnabled: boolean): Expression {
```

Comment:
There is too many updates, can we make this value default as false, so only true values are being passed from callers? Similar for below method 'parseTemplateExpression'

We just need to make sure all private methods have this as required parameter.

Reply:
@preetriti1 makes sense, addressed this in latest commit.